src/app/items/components/edit-item-modal/edit-item-modal.component.html

```diff
@@ -6,34 +6,54 @@ <h5>{{this.title}}</h5>
 </div>
 <div class="modal-body text-center">
   <form class="text-center" [formGroup]="editItemForm" autocomplete="on">
-    <div class="md-form md-outline">
-      <input mdbInput mdbValidate type="text" id="nameForm" class="form-control" [(ngModel)]="itemToEdit.name"
-        formControlName="nameForm">
-      <label for="nameForm">Name</label>
-      <mdb-error *ngIf="nameForm.invalid && (nameForm.dirty || nameForm.touched)">Name is invalid</mdb-error>
-    </div>
+    <div class="row">
+      <div class="col-md-7">
+        <div class="md-form md-outline">
+          <input mdbInput mdbValidate type="text" id="nameForm" class="form-control" [(ngModel)]="itemToEdit.name"
+                 formControlName="nameForm">
+          <label for="nameForm">Name</label>
+          <mdb-error *ngIf="nameForm.invalid && (nameForm.dirty || nameForm.touched)">Name is invalid</mdb-error>
+        </div>
 
-    <div class="md-form md-outline">
+        <div class="md-form md-outline">
       <textarea class="md-textarea form-control" rows="3" id="descriptionForm" type="text" length="120" mdbCharCounter
-        mdbValidate mdbInput [(ngModel)]="itemToEdit.description" formControlName="descriptionForm"></textarea>
-      <label for="descriptionForm">Description</label>
-      <mdb-error *ngIf="descriptionForm.invalid && (descriptionForm.dirty || descriptionForm.touched)">Description is
-        invalid</mdb-error>
-    </div>
-
-    <div class="md-form md-outline">
-      <input mdbInput mdbValidate type="number" id="categoryIdForm" class="form-control"
-        [(ngModel)]="itemToEdit.categoryId" formControlName="categoryIdForm">
-      <label for="categoryIdForm">Category ID</label>
-      <mdb-error *ngIf="categoryIdForm.invalid && (categoryIdForm.dirty || categoryIdForm.touched)">Category Id is
-        invalid</mdb-error>
+                mdbValidate mdbInput [(ngModel)]="itemToEdit.description" formControlName="descriptionForm"></textarea>
+          <label for="descriptionForm">Description</label>
+          <mdb-error *ngIf="descriptionForm.invalid && (descriptionForm.dirty || descriptionForm.touched)">Description
+            is invalid
+          </mdb-error>
+        </div>
+        <div class="md-form md-outline">
+          <input mdbInput mdbValidate type="number" id="categoryIdForm" class="form-control"
+                 [(ngModel)]="itemToEdit.categoryId" formControlName="categoryIdForm">
+          <label for="categoryIdForm">Category ID</label>
```

Comment:
show the categories in mdb select or some dropdown and not as IDs.

- [ ] select category and not id

Reply:
@TalShperling your job my friend